scripts/docker/CC-Devel/bin/install_llvm.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
+
+function cleanup() {
+    echo "Cleaning up LLVM temporaries."
+    if  [ -n "${LLVM_BUILD_DIR}" ]; then
+        rm -rf "${LLVM_BUILD_DIR}"
+    fi
+}
+
+trap cleanup EXIT
+
+set -e
+
+function usage() {
+    echo "${0}"
+    echo "  -h  Print this usage information. Optional."
+    echo "  -l  LLVM version. Mandatory. For example '3.8.0'."
+}
+
+while getopts ":hl:" OPTION; do
+    case ${OPTION} in
+        h)
+            usage
+            exit 0
+            ;;
+        l)
+            LLVM_VERSION="${OPTARG}"
+            ;;
+        *)
+            usage
+            exit 1
+            ;;
+    esac
+done
+
+if [[ -z "${LLVM_VERSION}" ]]; then
+    echo "LLVM version should be defined." >&2
+    usage
+    exit 1
+fi
+
+LLVM_INSTALL_DIR="/opt/llvm"
+LLVM_BUILD_DIR="/tmp/llvm"
+LLVM_MIRROR="http://releases.llvm.org"
```

Comment:
Upcoming changes to CodeCompass might (some of mine **will**) require a developer version of LLVM, rather than a release. Could you please add, if not a script that checks out from GitHub, but at least a script I could use to tell the docker creation system to "use ODB source from `<directory on local machine>`, and same with LLVM?